css/styles.css

```diff
@@ -0,0 +1,84 @@
+@font-face {
+    font-family: 'Noto Sans';
+    src: url(../fonts/NotoSans-Regular.ttf);
+    font-weight: normal;
+    font-style: normal;
+}
+
+@font-face {
+    font-family: 'Noto Sans';
+    src: url(../fonts/NotoSans-Bold.ttf);
+    font-weight: bold;
+}
+
+body {
+    color: #424242;
+    font-family: 'Noto Sans', sans-serif;
```

Comment:
I agree with Devon we shouldn't push the Kolibri styling too much, so perhaps just 
```
    font-family: 'sans-serif';
```
looks pretty decent on mac

<img width="505" alt="screen shot 2018-04-20 at 1 10 14 am" src="https://user-images.githubusercontent.com/163966/39031694-c105efca-4437-11e8-903f-092a4ac0aee3.png">

(not sure what some more specific sans fonts we could choose that are available cross platform)

Reply:
@ivanistheone's advice sounds good, but just another possibility...

Perhaps the font face is maybe the one place where perhaps we should mimic Kolibri, because:

1. It's highly unlikely that Kolibri will switch fonts
2. It will already have been downloaded by Kolibri
3. Multiple fonts on the same page can be visually jarring
4. Noto has excellent cross-language support

So you could potentially leave this line as-is (with `Noto Sans` default with `sans-serif` fallback), and then just don't explicitly include the font files.

Reply:
I think these are all good points, and particularly (3) from a purely user point of view. And the other point I want to raise is that currently, the font face is perhaps the most salient piece of styling and I think the one most responsible for making the theme look and feel like Kolibri.

However, the case for not emulating Kolibri due to possibility of updating internal styles/colors is still a valid point. So how about we aim for the following:

Emulate a "neutral" Kolibri look-and-feel, in particular, one that would be resilient to changes to Kolibri styling and could stand on its own with Kolibri's current styling or expected changes to it. Like a piece of furniture that fits the current decor but can still belong if we renovate the place (my old [light wooden IKEA coffee table](http://sweethings.net/wp-content/uploads/2016/02/Mesa-antes.jpg) comes to mind). So, no particular color hue (whites and greys OK) or bold strong opinionated features.

So, concretely, here are some suggestions to enact that idea:

- line-height: same as Kolibri
- letter-spacing: same as Kolibri
- font sizes: same as Kolibri
- font color: same as Kolibri
- link color/styling: non-Kolibri generic (underline, bold, or blue)
- button: non-Kolibri generic (material white)

Screenshot:

![image](https://user-images.githubusercontent.com/159687/39208622-6160ba6c-47b8-11e8-8a12-09ab5d41ba5f.png)

(The only things different from the screenshots in the original incantation of this PR are the link color, button color, and button hover (no effect).)

I also want to caveat that I have no formal design education, so please chime in if you thoughts.

Reply:
Looks like there are iframe restrictions which doesn't make this straightforward. Just referencing the font as `font-family: 'Noto Sans';` inside the iframe doesn't work even if that font is available in the parent window.

What I think is possible is including something like this in the iframe's stylesheet:

```
@font-face {
    font-family: 'Noto Sans';
    src: url(/path/to/font/on/kolibri/server/notosans.ttf);
}
```

... and I think that's going to require some changes to Kolibri server too, to set the proper access control rules so the iframe can access it.

For now I'm just falling back to the default browser sans serif font as @ivanistheone suggested.